src/app/Login/page.module.css

```diff
@@ -22,35 +26,108 @@
 }
 
 .welcomeBackcontainer {
-  margin: 100px;
-  text-align: center;
-  font-size: 50px;
-  text-align: center;
-  color: #ffffff;
-  margin-bottom: 100px;
-}
-
-.loginPageContainer {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 100vh;
-  width: 100vw;
+  margin: 3rem;
+  font-size: 3rem;
+  margin-bottom: 3rem;
 }
-
 .lgnbttncontainer,
 .signbttncontainer {
   display: flex;
   justify-content: center;
   align-items: center;
-  margin-top: 20px;
+  margin-top: 1rem;
 }
 
 .Err {
   color: #e74c3c;
   font-size: 14px;
-  margin-top: 5px;
+  margin-top: 1rem;
   font-weight: bold;
   display: flex;
   align-items: center;
 }
+@keyframes shake {
+  0% {
+    transform: translateX(0);
+  }
+  25% {
+    transform: translateX(-5px);
+  }
+  50% {
+    transform: translateX(5px);
+  }
+  75% {
+    transform: translateX(-5px);
+  }
+  100% {
+    transform: translateX(0);
+  }
+}
+
+.Err.shake {
+  animation: shake 0.2s ease; /* Adjust duration and timing as needed */
+}
+
+@media (min-width: 300px) and (max-width: 380px) {
+  .loginPageContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+  .welcomeBackcontainer {
+    margin: 2rem;
+    text-align: center;
+    font-size: 1.5rem;
+    margin-bottom: 2rem;
+    margin-top: 1rem;
+  }
```

Comment:
Rename, this is used on the `<h1>` tag and its inaccurate to call this class a container. also the font-size is too small here on smaller devices, `2-3rem` might be appropriate but experiment.

Also double check the margins here, `margin: 2rem` means you define top, right, bottom, and left margins to `2rem` but then your also redefining it on lines 81 and 82. 

This link will show and has a playground on how margins in CSS act
https://developer.mozilla.org/en-US/docs/Web/CSS/margin